dss/api/subscriptions_v1.py

```diff
@@ -45,7 +45,7 @@ def get(uuid: str, replica: str):
     return jsonify(source), requests.codes.okay
 
 
-@security.assert_security(method='read', groups=['dbio', 'public'])
+@security.assert_security(method='group', groups=['dbio', 'public'])
```

Comment:
was `sub_v1:put` missed? or intentionally left to `create`?

Reply:
`create` is the same as `group` under the hood